packages/create-gatsby/src/index.ts

```diff
@@ -323,7 +324,7 @@ ${c.bold(`Thanks! Here's what we'll now do:`)}
   `)
 
   console.log(`Start the local development server with\n
-  ${c.magenta(`${runCommand} develop`)}
+  ${c.magenta(`${pm} start`)}
```

Comment:
Why start? Did you got any feedback? I'm used to `yarn develop`. `yarn start` for me indicates production code :shrug: `gatsby build && gatsby serve` I might be the only one. Mostly curious on the why.

```suggestion
  ${c.magenta(`${pm} start`)}
```

Reply:
I also much prefer `develop`/`dev` over `start`, same reason as Ward!

Reply:
Because unlike yarn, npm won't let you do something like `npm develop`, as it won't run arbitrary scripts like that, and instead have to be run as `npm run develop`. The `start` script is a special case which can be run as `npm start`

Reply:
Happy to change back, it's just a simpler command

Reply:
I'll remove it from this PR and we can discuss later